core/src/main/java/feign/Contract.java

```diff
@@ -312,4 +315,27 @@ protected boolean processAnnotationsOnParameter(MethodMetadata data, Annotation[
       return result;
     }
   }
+
+  class DelegatingContract implements Contract, Param.Expander.Factory {
```

Comment:
Hi @spencergibb . My thought was that since Feign is Java 6-based I didn't want to change the `Contract` interface and break downstream subclasses.